index.html

```diff
@@ -21,7 +21,7 @@
     - The freeCodeCamp Team
   -->
 
-  <p class="paragraph">Hello World, we are software developers!</p>
+  <p class="paragraph">Hello World, we are software developers!!</p>
 
   <div id="main">
```

Comment:
Please use the `main` tag instead of a `div` tag to wrap up your code so it can be semantic.